diffkemp/simpll/passes/CalledFunctionsAnalysis.h

```diff
@@ -38,7 +38,9 @@ class CalledFunctionsAnalysis
     void collectCalled(const Function *Fun, Result &Called);
     /// Looks for functions in a value (either a function itself, or a composite
     /// type constant).
-    void processValue(const Value *Val, Result &Called);
+    void processValue(const Value *Val,
+                      Result &Called,
+                      std::set<const Value *> &Processed);
```

Comment:
Could we make `Processed` a private class member?

Reply:
We certainly can. That's a much cleaner solution, thank you.